docker-compose.yml

```diff
@@ -131,4 +132,3 @@ networks:
     main:
 volumes:
     vendor:
```

Comment:
I think with your above update we can also get rid of the volume definition here?

Reply:
Thanks - I must have missed that.